components/feature/sitepermissions/src/main/java/mozilla/components/feature/sitepermissions/SitePermissionsFeature.kt

```diff
@@ -188,6 +206,28 @@ class SitePermissionsFeature(
         )
     }
 
+    @SuppressLint("VisibleForTests")
+    private fun createPromptForMicrophonePermission(
+        context: Context,
+        uriString: String,
+        buttons: List<DoorhangerPrompt.Button>
+    ): DoorhangerPrompt {
+
+        val title = context.getString(R.string.mozac_feature_sitepermissions_microfone_title, uriString)
+        val optionTitle = context.getString(R.string.mozac_feature_sitepermissions_option_microphone_one)
+        val drawable = ContextCompat.getDrawable(context, R.drawable.mozac_ic_microphone)
+
+        val option1 = RadioButton(optionTitle).apply { checked = true }
```

Comment:
"option1" ?

Reply:
Naming is hard XD. I will change it :)

In the mock-ups they have a fine-grained microphone list, but right now GV is only providing us one (the default microphone) I guess in the future they are going to provide that.  
![image](https://user-images.githubusercontent.com/773158/53032698-5974ff80-343d-11e9-8934-ddebf8362f06.png)

Reply:
Is this also the case if there are external microphones available?

Reply:
I'm not quite sure, I'm just guessing from what I saw in the [mock-ups](https://github.com/mozilla-mobile/fenix/issues/332). They didn't include a mock-up with external microphones apart from the headset one.